### PR TITLE
Add ERC: Transaction risk monitoring

### PR DESCRIPTION
**Pull request by Flyme10086**
This ERC proposes a standardized framework for identifying, monitoring, and regulating high-risk transactions on the Ethereum network. The framework includes a set of criteria and a procedural approach for classifying transactions as high-risk based on factors such as transaction amount, frequency, associated addresses, and patterns indicative of fraudulent or illicit activities. The aim is to enhance network security and integrity, prevent illegal activities, and maintain compliance with global regulatory standards.

### CODE REVIEW

File: ERCS/erc-7688.md [hunk 1]
```diff
@@ -0,0 +1,75 @@
+---
+eip: 7688
```
Comment (abcoathup) [inline ERCS/erc-7688.md:2]:
```suggestion
eip: 7606
```
Assigning next sequential EIP/ERC/RIP number.
Numbering changed to sequential from 7500 and is no longer the PR number.

Please also update the filename.

File: ERCS/erc-7688.md [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+title: Ethereum RiskWatch Guide
```
Comment (abcoathup) [inline ERCS/erc-7688.md:3]:
```suggestion
title: Transaction risk monitoring
```
Suggest using a more descriptive title.  This is an ERC so doesn't need to include the word Ethereum.

**Comment by lex-node** [comment]
no, absolutely not--kill it with fire, nuke it from orbit--I don't care 

you can't have one "standardized framework" for this because every country's laws are different and they change all the time; moreover, this should never be enshrined in the protocol, which should remain credibly neutral (i.e., not following any particular country's laws or "risk" standards) 

where would you draw the line? might as well add in China's social credit score while you're at it

**Comment by pcfreak30** [comment]
Two words.

HELL. NO.

Kudos.

**Comment by 0xDreamWeaver** [comment]
Eat shit

**Comment by ControlCplusControlV** [comment]
if you're going to make a honeypot atleast do it on your main github coward (and if this is your main you should be ashamed lmao)

**Comment by pcfreak30** [comment]
> I think this reintroduces some of the worst aspects of the old system while creating its own new set of problems unique to the new environment.
> 
> Catering to regulators and gatekeepers (which ones?) over financial freedom of the individual undermines the purpose of building an open source financial system within distributed systems at great expense and complexity.
> 
> I think Vitalik’s proposal to use zero knowledge proofs to prove things like “I am not a North Korean hacker” can achieve some of these goals without revealing personal information or removing sovereignty from the individual.

TBH, even that is a slippery slope. That is a form of digital ID (`soul bound token`) to prove innocence, which said another way `guilty until proven innocent`. This will always be a fight, but the pragmatic or progressive view will get abused by different groups to discriminate with the tools being made available.

**Comment by abcoathup** [comment]
@lex-node, @pcfreak30, @0xDreamWeaver, @ControlCplusControlV & @miller46 

For discussions on the proposed ERC please use the Eth Magicians discussions thread:
https://ethereum-magicians.org/t/ethereum-high-risk-transaction-surveillance-guidelines/18415

**Comment by geet-eth** [comment]
fuck no

File: ERCS/erc-7606.md [hunk 3]
```diff
@@ -0,0 +1,75 @@
...
+
+Step Four: Evaluation of Suspicious Address Set
+
+- Suspiciousness Evaluation: Assess the suspiciousness of each address using a weighted coefficient of frequency sorting, transaction frequency, transaction amount, and correlation.
+
+This simplified specification outlines key heuristic rules, providing a concise overview of the process for tracking high-risk transactions on Ethereum.
+
+## Rationale
+
+The motivation for this standard is to provide a comprehensive framework for monitoring high-risk transactions, balancing the need for ordinary users' privacy with the necessity of preventing criminal misuse. Detailed design motivations and alternative considerations will be provided in subsequent versions of this EIP.
```
Comment (SamWilsn) [inline ERCS/erc-7606.md:55]: This paragraph reads like it belongs in the Motivation section. The Rationale section should explain technical choices made within the proposal.

> Motivation: "Why do we need a shed?"
> Rationale: "Why is the shed red?"

File: ERCS/erc-7606.md [hunk 4]
```diff
@@ -0,0 +1,75 @@
...
+
+No backward compatibility issues found.
+
+## Test Cases
+
+TBD
+
+## Reference Implementation
+
+TBD
```
Comment (SamWilsn) [inline ERCS/erc-7606.md:67]:
```suggestion
```

These sections are optional.

File: ERCS/erc-7606.md [hunk 5]
```diff
@@ -0,0 +1,75 @@
...
+
+## Security Considerations
+
+The security considerations of this standard are under discussion.
```
Comment (SamWilsn) [inline ERCS/erc-7606.md:71]:
```suggestion
<!-- TODO -->
```

If you use an HTML-style comment, the linter will make sure you replace it before you move to Review.

File: ERCS/erc-7606.md [hunk 6]
```diff
@@ -0,0 +1,75 @@
+---
+eip: 7606
+title: Transaction risk monitoring
+description: Establishing a framework for monitoring and regulating high-risk
```
Comment (SamWilsn) [inline ERCS/erc-7606.md:4]: Your description isn't a complete sentence, nor does it really enough information that wasn't already in the title.

**Review by SamWilsn — CHANGES_REQUESTED** [review]
This proposal doesn't really include enough detail to be implementable, nor does it do a good job of explaining what parties would need to implement it. I can't tell if this is meant to be implemented by clients like geth, or if this is an off-chain oracle service for smart contracts.

**Pull request closed without merging**